### PR TITLE
added target switcher and updated coords

### PR DESCRIPTION
**Pull request by scottaames**

### CODE REVIEW

**Review by jkaufy — APPROVED** [review]
looks good.

File: app/src/components/container/MapContainer.jsx [hunk 1]
```diff
@@ -13,11 +13,29 @@ export default class MapContainer extends Component {
...
+    let oldContainer = document.getElementById("map-container");
+    let parent = oldContainer.parentNode;
+    let newContainer = document.createElement("div");
+    parent.removeChild(oldContainer);
+    newContainer.setAttribute("id", "map-container");
+    parent.appendChild(newContainer);
+
+    let map = new AstroMap("map-container", this.props.target, {});
+    new Projection().addTo(map);
+    new MousePosition({
```
Comment (kaitlyndlee) [inline app/src/components/container/MapContainer.jsx:45]: I am not sure what happened to master, but we were adding a scale control to the map. It looks like it got deleted somehow, not by your PR. Could you add in adding the control? Should be: new L.Control.Scale({ imperial: false }).addTo(map);

File: app/src/js/mousePosition.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ export default L.Control.MousePosition = L.Control.extend({
     separator: ", ",
     numDigits: 5,
     prefix: "",
-    targetPlanet: ""
+    targetPlanet: "",
+    lngFirst: true
```
Comment (kaitlyndlee) [inline app/src/js/mousePosition.js:25]: I think the map by default needs to show lat first, unless we swapped the coordinates somewhere. Lat should be between -90, 90

File: app/src/js/mousePosition.js [hunk 3]
```diff
@@ -142,13 +143,13 @@ export default L.Control.MousePosition = L.Control.extend({
       : lat + this.options.separator + lng;
```
Comment (kaitlyndlee) [inline app/src/js/mousePosition.js:143]: looks like here we are setting lat first, or is that a wrong assumption?

Comment (kaitlyndlee) [reply app/src/js/mousePosition.js:143]: Nevermind I didn't see the whole if branch.

File: app/src/js/mousePosition.js (hunk 2 above)
Comment (kaitlyndlee) [reply app/src/js/mousePosition.js:25]: Nevermind!

**Review by cgm78 — APPROVED** [review]

**Review by kaitlyndlee — APPROVED** [review]

**Pull request merged**